### DISCUSSION TITLE
Update README

### DISCUSSION THREAD

**Discussion by thuongsugar** [SWHL/RapidVideOCR#58] [Q&A] [answered]
I followed **readme.md**, I did **pip install rapid_videocr**, it is old version from 2023 with **version 2.1.3**, but on **pypi.org** it is **2.2.13** with the statement **pip install rapid-videocr**. Basically, nothing has changed much, right?

**Comment by SWHL** [comment, marked as answer]
There is no difference between the following two installation methods:
```bash
pip install rapid_videocr

pip install rapid-videocr
```
